api/controllers/CheckUploadedDocumentsController.js

```diff
@@ -14,20 +17,32 @@ const CheckUploadedDocumentsController = {
             if (uploadedFileData.length === 0) {
                 throw new Error('No files uploaded');
             }
-            uploadedFileData.forEach(async (uploadedFile) => {
-                await UploadedDocumentUrls.create(
-                    CheckUploadedDocumentsController._dbColumnData(
-                        uploadedFile,
-                        req
+            for (let i = 0; i <= uploadedFileData.length; i++) {
+                const endOfLoop = i === uploadedFileData.length;
```

Comment:
this is minor, something to keep in mind for next time, so no need to change for now. 

`.length` is base 1 (not base 0) - so your `endOfLoop` if statement will occur after all of the files have been iterated through, if that makes sense? just double check that's all working as you intended!

Reply:
Yeah, it's a bit confusing but the idea is that after the data in the loop is finished, as in, all the data has been processed, then run the `_checkDocumentCountAndPaymentDetails ` function.

I guess I could also run it on `ploadedFileData.length -1` as well 🤔